workspace/notebook/py_hzn_standardized_s78_tables.json

```
@@ -136,443 +136,525 @@
 					"insert_count = 0\r\n",
 					"update_count = 0\r\n",
 					"delete_count = 0\r\n",
-					"error_message=''"
+					"error_message='None'\r\n",
+					""
 				],
 				"execution_count": null
 			},
 			{
 				"cell_type": "code",
+				"source": [
+					"def get_spark_df(table_name: str):\r\n",
+					"    try:\r\n",
+					"        df_filtered = None\r\n",
+					"        error_message = None\r\n",
+					"        end_exec_time = None\r\n",
+					"        df = spark.table(table_name)\r\n",
+					"        max_date = df.agg(F.max(\"expected_from\")).collect()[0][0]\r\n",
```

if you're missing the expected_from this will blow up or where it is null
is standardise layer always provding it ? cause in horizon you have exception where it doesn't come. You should validate the column exists and else fail fast with a message